UMI3D-Browser-Desktop/Assets/Dependencies/UMI3D SDK/ClientDevelopmentKit/Collaboration/Runtime/Networking/UMI3DCollaborationClientServer.cs

```diff
@@ -171,6 +171,8 @@ public static async void Connect(RedirectionDto redirection, Action<string> fail
                     UMI3DWorldControllerClient wc = worldControllerClient?.Redirection(redirection) ?? new UMI3DWorldControllerClient(redirection);
                     if (await wc.Connect())
                     {
+                        await wc.DownloadWorldLibraries();
```

Comment:
Why isn't this part of the connect method ?

Reply:
while resolving another comment, i now have a private variable set before (and needed)